d/downshift/build_info.json

```diff
@@ -0,0 +1,15 @@
+{
+    "maintainer":"vinodk99",
+    "package_name" : "downshift",
+    "github_url": "https://github.com/downshift-js/downshift",
+    "version": "master",
```

Comment:
@onkar82kubal  This should be version for which you created and tested build script. Same comment for your other PR.

Other things look good

Reply:
Fixed